schemas/disease_target_evidence.json

```diff
@@ -1488,6 +1482,52 @@
       "minItems": 1,
       "uniqueItems": true
     },
+    "assays": {
+      "type": "array",
+      "description": "Assays used in the study",
+      "items": {
+        "type": "object",
+        "description": "Description and result of one assay",
+        "properties": {
+          "shortName": {
+            "type": "string",
+            "description": "Short name of the assay.",
+            "examples": [
+              "CellTiter-Glo"
+            ]
+          },
+          "description": {
+            "type": "string",
+            "description": "Description of the assay",
+            "examples": [
+              "Homogeneous method of quantifying the presence of ATP, an indicator of metabolically active cells, and using this as a proxy to determine the number of viable cells."
+            ]
+          },
+          "isHit": {
```

Comment:
If all assays were negative for an evidence, would the evidence still be indicative of an association?

Reply:
If that's what the valiation result shows, it would still be in the evidence set. It would be a strong negative evidence. If I got your question right.